keras/utils/vis_utils.py

```diff
@@ -61,7 +61,7 @@ def model_to_dot(model,
     if isinstance(model, Sequential):
         if not model.built:
             model.build()
-        model = model.model
+        model = model
```

Comment:
This line is now useless. I think the line above (build) might be useless too, please try it.

Reply:
Deleting the entire ``if isinstance`` block worked locally with

```
import keras

model = keras.models.Sequential()
model.add(keras.layers.Dense(10, input_dim=50))

keras.utils.plot_model(model, "sequential.png")
``

Reply:
So does this:

``
import keras
from keras.layers import Dense
from keras.models import Sequential

model = Sequential()
model.add(Dense(32))
model.add(Dense(32))
#    model.build((None, 500))
keras.utils.plot_model(model, "sequential.png", show_shapes=True)
``

Reply:
The plot without the build is a little funky:

![sequential](https://user-images.githubusercontent.com/33430930/40333324-c6da6b04-5d0c-11e8-8120-fc5f8ed206c8.png)